dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityCollectionParentIT.java

```diff
@@ -0,0 +1,218 @@
+                                        .matchCollectionEntry(col2.getName(), col2.getID(), col2.getHandle())))));
+
+    }
+
+    @Test
+    public void itemYOwningCollectionTest() throws Exception {
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/items/" + itemY.getID() + "/owningCollection"))
+                        .andExpect(jsonPath("$", is(CollectionMatcher.matchCollectionEntry(col1.getName(), col1.getID(),
+                                                                                           col1.getHandle()))))
+                        .andExpect(jsonPath("$", Matchers
+                            .not(is(CollectionMatcher
+                                        .matchCollectionEntry(col2.getName(), col2.getID(), col2.getHandle())))));
+    }
+
+    @Test
+    public void itemZOwningCollectionTest() throws Exception {
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/items/" + itemZ.getID() + "/owningCollection"))
+                        .andExpect(jsonPath("$", is(CollectionMatcher.matchCollectionEntry(col2.getName(), col2.getID(),
+                                                                                           col2.getHandle()))))
+                        .andExpect(jsonPath("$", Matchers
+                            .not(is(CollectionMatcher
+                                        .matchCollectionEntry(col1.getName(), col1.getID(), col1.getHandle())))));
+
+    }
+
+    @Test
+    public void col1ParentCommunityTest() throws Exception {
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/core/collections/" + col1.getID() + "/parentCommunity"))
+                        .andExpect(jsonPath("$", is(CommunityMatcher
+                                                        .matchCommunityEntry(communityAA.getName(), communityAA.getID(),
+                                                                             communityAA.getHandle()))))
+                        .andExpect(jsonPath("$", not(is(CommunityMatcher.matchCommunityEntry(communityA.getName(),
+                                                                                             communityA.getID(),
+                                                                                             communityA.getHandle())))))
+                        .andExpect(jsonPath("$", not(is(CommunityMatcher.matchCommunityEntry(communityAB.getName(),
+                                                                                             communityAB.getID(),
+                                                                                             communityAB
+                                                                                                 .getHandle())))));
+
+    }
+
+    @Test
+    public void col2ParentCommunityTest() throws Exception {
```

Comment:
my suggestion is to remove this test as it is functionally equivalent to col1ParentCommunityTest